src/screen.cpp

```diff
@@ -0,0 +1,58 @@
+#include "../include/screen.h"
+
+// Constructor
+Screen::Screen(int width, int height, const std::string& title){
+  m_width = width;
+  m_height = height;
+  m_title = title;
+}
+
+Screen::~Screen(){
+  SDL_GL_DeleteContext(m_glContext);
+  SDL_DestroyWindow(m_window);
+  SDL_Quit();
+}
+
+// Screen initializer
+void Screen::Init(){
```

Comment:
I was kinda trying to offload the work done by the constructor (i don't even know why), but would making init private and calling it at the end of the constructor do?